paddle/inference/inference.cc

```diff
@@ -52,39 +52,15 @@ void InferenceEngine::LoadInferenceModel(const std::string& dirname) {
   }
 }
 
```

Comment:
This is to make the ordering of the `feed_var_names_` (also known as  `feed_target_names`) consistent with the  python side when using the save_inference_model function.  

Say you use code like `save_inference_model(MODEL_DIR, ["x", "y"], [avg_cost], exe, program)` in python. 

Then based on our code when prepending feed operators (because we first prepend "x", then "y"),  the order of feed ops in block.AllOps will be like [feed("y") feed("x")].

So by doing the insert thing on line 48, we can make the order of vector `feed_var_names_` to be ["x", "y"] which is consistent with the python side order. So that we you provide a `vector<Tensor> feeds`, you know the order is the same as the python `save_inference_model`.

Reply:
I see. Thanks~